Makefile

```diff
@@ -215,38 +213,14 @@ depend: init | $(ISTIO_OUT)
 $(ISTIO_OUT) $(ISTIO_BIN):
 	@mkdir -p $@
 
-depend.ensure: init
-
-# Target to update the Gopkg.lock with latest versions.
-# Should be run when adding any new dependency and periodically.
-depend.update: ${DEP} ; $(info $(H) ensuring dependencies are up to date...)
-	${DEP} ensure
-	${DEP} ensure -update
-
 # If CGO_ENABLED=0 then go get tries to install in system directories.
 # If -pkgdir <dir> is also used then various additional .a files are present.
-${DEP}:
-	unset GOOS && CGO_ENABLED=1 go get -u github.com/golang/dep/cmd/dep
-
 ${GOLINT}:
 	unset GOOS && CGO_ENABLED=1 go get -u github.com/golang/lint/golint
```

Comment:
btw we should probably not get golint that way either - I think bin/linter.sh use an image ...

Reply:
There were some other linter targets that I think have since been removed so this target (and the GOLINT variable) might now be unused (though I didn't search the other .mk files that are imported by this one).

Reply:
indeed
```
ldemailly-macbookpro:istio ldemailly$ git grep GOLINT
Makefile:GOLINT   := $(shell which golint 2>/dev/null || echo "${ISTIO_BIN}/golint" )
Makefile:${GOLINT}:
```
will drop it but I want to see all builds green first before touching the pr again and cancelling the inflight ones